src/js/game/LevelMVC/LevelView.js

```diff
@@ -682,58 +677,40 @@ module.exports = class LevelView {
     //make sure to render high for when moving up after placing a block
     var zOrderYIndex = position[1] + (facing === FacingDirection.Up ? 1 : 0);
     this.player.sprite.sortOrder = this.yToIndex(zOrderYIndex) + 5;
-    oldPosition = [Math.trunc((this.player.sprite.position.x + 18) / 40), Math.ceil((this.player.sprite.position.y + 32) / 40)];
-    newPosVec = [position[0] - oldPosition[0], position[1] - oldPosition[1]];
-
-    //change offset for moving on top of blocks
-    if (isOnBlock) {
-      yOffset -= 22;
-    }
```

Comment:
was 22, now 23 with your new helper `positionToScreen`. do we care?

Reply:
Ah, good catch.  I think they're close enough?  `23px` is on the left, `22px` is on the right:

![screen shot 2017-09-01 at 3 38 27 pm](https://user-images.githubusercontent.com/413693/29990043-26e29b8e-8f2c-11e7-8276-7c135074568b.png)

FWIW I measured the height of the side of a block and it's actually `20px` 🤷‍♀️.

Reply:
I can tell the difference. Seems fine to me.